src/contexts/AppContext.tsx

```diff
@@ -40,8 +40,11 @@ export const AppContextProvider = ({ children }: PropsWithChildren) => {
   const [appModal, setAppModal] = useState<AppModalValues>({
     isOpen: false,
   });
+  const [refreshTrigger, setRefreshTrigger] = useState<string>('');
```

Comment:
Didn't understand the purpose of this state variable, could we add a comment? or rename it to `refreshDrops`? or simply refresh the page when upload is completed? At first i thought it was to refresh the page.

Reply:
removing this